chain/cmd/bandoracled2/event.go

```diff
@@ -40,9 +42,10 @@ func GetRawRequests(log sdk.ABCIMessageLog) ([]rawRequest, error) {
 		}
 
 		reqs = append(reqs, rawRequest{
-			dataSourceID: otypes.DataSourceID(dataSourceID),
-			externalID:   otypes.ExternalID(externalID),
-			calldata:     calldataList[idx],
+			dataSourceID:   otypes.DataSourceID(dataSourceID),
```

Comment:
I don't think this field is not needed anymore?

Reply:
Still used in handler